src/internal/components/dropdown-status/index.tsx

```diff
@@ -14,7 +12,7 @@ import styles from './styles.css.js';
 
 export { DropdownStatusProps };
 
-export interface DropdownStatusPropsExtended extends DropdownStatusProps {
+interface UseDropdownStatusProps extends DropdownStatusProps {
```

Comment:
1. Never used outside this file, so no `export` needed
2. It is used for a hook arguments, so let's name it correctly